first.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML page with links</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+</head>
+
+<body>
+
+  <div class="jumbotron text-center">
+    <h1>Introduction to HTML, Javascript & CSS</h1>
+    <p>My first exersice!</p>
+  </div>
+
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-4">
+        <h3>Link for local directory</h3>
+        <p>Desktop directory</p>
+        <p><a target="_lank" href="New/">Directory</a></p>
+
+      </div>
+	  <!--This is open on explored-->
+      <div class="col-sm-4">
+        <h3>Link for text file from my computer</h3>
+        <p>Text file</p>
+        <p><a target="_lank" href="new.txt">Text File</a></p>
+     
+	  </div>
+      <div class="col-sm-4">
+        <h3>Link for html page from my computer</h3>
+        <p>HTML created on my computer by me</p>
+        <p><a target="_lank" href="1.html">HTML page</a></p>
```

Comment:
typo: "_blank"

Reply:
I'm sorry I made a mistake, it must be "_blank". I'll correct that right away.

Reply:
I fixed it.